packages/lsp-tools/test/doenet-auto-complete.test.ts

```diff
@@ -166,6 +166,25 @@ describe("AutoCompleter", () => {
             `);
         }
     });
+    it("Can suggest completions after a `<`", () => {
```

Comment:
This tests makes me think that after typing "<" in the editor, I should get some completions. However I don't get completions after typing just "<" inside a `<codeEditor>` tag (in test-viewer dev server) or inside the editor from the dev server in lsp-tools.

Reply:
Really? I need to look into that then...It gives suggestions to me when testing out in `packages/codemirror`.

Reply:
Tried it out more.  It turns out it doesn't work if my cursor is at the end of the file, but it works otherwise. I was always testing at the end of the file, so it wasn't working.

Reply:
Also, if at end of file and have an unclosed open tag, then `<` does not suggest the closing tag.